tests/test_wagon.py

```diff
@@ -105,12 +97,6 @@ def test_download_missing_path(self):
             wagon._download_file(TEST_TAR, 'x/file')
         assert 'No such file or directory' in str(ex.value)
 
-    @pytest.mark.skipif(wagon.IS_WIN, reason='Irrelevant on Windows')
-    def test_download_no_permissions(self):
```

Comment:
this (and the one below, `test_tar_no_permissions`) won't work when ran as root, and I don't like tests that work differently depending on how do you run them.
Also, I don't think we need tests that check that it's impossible to write to locations that are not writable..